coordinator/index.ts

```diff
@@ -0,0 +1,2 @@
+export * as getResourcesOfType from "./src/services/translation/common/getResourcesOfType"
+export {convertFhirMessageToSignedInfoMessage} from "./src/services/translation/request"
```

Comment:
expose coordinator apis to use in other projects, these are consumed by referenced pact project